src/VirtualClient/VirtualClient.Actions.UnitTests/Wrathmark/WrathmarkTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace VirtualClient.Actions
+{
+    using System.Diagnostics;
+    using System.IO;
+    using System.Reflection;
+
+    using NUnit.Framework;
+
+    using VirtualClient.Actions.Wrathmark;
+
+    [TestFixture]
+    [Category("Unit")]
+    public class WrathmarkTests
```

Comment:
What is this for? In unit tests we almost never touches actual file systems, except for parser tests.  For things that touches the file system we use the integrationTests project

Reply:
It's a base class for Wrathmark unit tests. I've updated the name, made it protected, and added XML documentation for future maintainers.

Reply:
For the file system, there are other tests that use the file system. Can update to not use the example so it's all self contained as a constant, or use the convention established with the Examples folder. If the latter, I'm confused why the Examples folder exists in the Unit Tests project

Reply:
Yes Example folder are only meant for parsers in unit test and only exception we do for accessing filesystem during unit test. It's ok, just we generally don't access filesystem for anything else